users/models.py

```diff
@@ -141,7 +143,9 @@ class User(AbstractBaseUser, TimestampedModel, PermissionsMixin):
     USERNAME_FIELD = "username"
     REQUIRED_FIELDS = ["email", "name"]
 
-    username = models.CharField(unique=True, default=generate_username, max_length=26)
+    username = models.CharField(
+        unique=True, default=generate_username, max_length=USERNAME_MAX_LEN
```

Comment:
Could you remove the `default` here too so we're sure that users don't get a `ulid` username by accident?